app/partials/recover-funds.jade

```diff
@@ -75,6 +75,14 @@
         span.help-block 
           div(ng-show="recoveryForm.confirmation.$touched")
             p(ng-show="recoveryForm.confirmation.$error.isValid" translate="NO_MATCH")
+    .flex-center.flex-end.mtm.mbl
+      input#agreement_accept.pull-right(
+        name="agreement"
+        type="checkbox"
+        ng-model="fields.acceptedAgreement"
```

Comment:
Right, the form won't be valid unless the checkbox is checked. So you just need to look for that and not worry about `fields.acceptedAgreement`.